crates/ra_assists/src/fill_match_arms.rs

```diff
@@ -119,6 +123,105 @@ mod tests {
             "#,
         );
     }
+
+    #[test]
+    fn test_fill_match_arm_refs() {
+        check_assist(
+            fill_match_arms,
+            r#"
+            enum A {
+                As,
+                Bs,
+                Cs(String),
+                Ds(String, String),
+                Es{x: usize, y: usize}
```

Comment:
Let's use enum with just one variant for this test? Match ergonomics should be orthogonal to variants generation, and having smaller tests is useful!